### DISCUSSION TITLE
New payments methods

### DISCUSSION THREAD

**Discussion by EnderHosting** [Paymenter/Paymenter#284] [Ideas]
I suggest add Google pay method

**Comment by GreenArrowlol** [comment]
what do you mean google pay? are you talking about UPI? 
there's no gateway called google pay.

**Comment by GomorrhaDev** [comment]
> I suggest add Google pay method

Documentation: https://developers.google.com/pay/api

**Comment by MauroSupera** [comment]
Hi, I would like to suggest to add MercadoPago's gateway as an extension it would help a lot ❤️ great panel

**Reply by GomorrhaDev** [reply]
Create a new suggestion. It has nothing to do with Google Pay, if I have understood it correctly.

**Comment by Griffeng** [comment]
Google pay is not something you can add standalone, the only way to have Google pay is to get it with another payment service provider.

**Comment by 65** [comment]
You can add on Google Pay via Stripe https://docs.stripe.com/google-pay
